proto/buf.yaml

```diff
@@ -2,10 +2,11 @@ version: v1
 name: buf.build/cosmos/ibc
 deps:
   # see: (https://github.com/cosmos/cosmos-sdk/tree/main/proto#sdk-x-buf)
-  - buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176
-  - buf.build/cosmos/cosmos-proto:1935555c206d4afb9e94615dfd0fad31
-  - buf.build/cosmos/gogo-proto:bee5511075b7499da6178d9e4aaa628b
-  - buf.build/googleapis/googleapis:783e4b5374fa488ab068d08af9658438
+   - buf.build/cosmos/cosmos-sdk # TODO to pin
+   - buf.build/cosmos/cosmos-proto:1935555c206d4afb9e94615dfd0fad31
+   - buf.build/cosmos/gogo-proto
+   - buf.build/googleapis/googleapis:8d7204855ec14631a499bd7393ce1970
+   - buf.build/cosmos/ics23:b1abd8678aab07165efd453c96796a179eb3131f
```

Comment:
We should pin to versions for sdk and gogo-proto. Is it okay to use latest tags @julienrbrt?

Reply:
Yes, it's best to pin so you don't have to worry about possible proto breaking changes.
If some buf dependencies need to be upgraded because of an SDK upgrade, we will mention it in our UPGRADING.md.

Reply:
Awesome, thanks! Updated :)